src/App.js

```diff
@@ -1,25 +1,64 @@
-import logo from './logo.svg';
 import './App.css';
+import contactsData from './contacts.json';
+import { useState } from "react";
 
 function App() {
+
+  const [contactsToDisplay, setContactsToDisplay] = useState(contactsData)
```

Comment:
Com isso o seu state vai usar os 5 primeiros

```suggestion
  const [contactsToDisplay, setContactsToDisplay] = useState(fiveContacts )
```